### DISCUSSION TITLE
Enclosure/chamber temp on Job Status panel?

### DISCUSSION THREAD

**Discussion by Evo589** [KlipperScreen/KlipperScreen#432] [Q&A]
Hello,

I have an enclosure temperature sensor set up in klipper and I see it in the Main Menu panel. However. when printing, the Job Status panel does not show it. It only shows hotend and bed temp  Is there a way to add the enclosure temp to the Job Status panel.

**Comment by alfrix** [comment]
currently you must use titlebar_items, since there is limited space the amount of items allowed depends on the screen resolution
https://klipperscreen.readthedocs.io/en/latest/Configuration/#printer-options
